examples/daal/cpp_sycl/source/logistic_regression/logistic_regression_online.h

```diff
@@ -0,0 +1,265 @@
+        _solver->parameter.batchSize            = batchRowCount;
+        _solver->parameter.conservativeSequence = daal_dm::SyclHomogenNumericTable<Fptype>::create(1, 1, daal_dm::NumericTable::doAllocate, 0.0);
+    }
+
+    // Sets the parameters of current iteration of learning process
+    void setIterationParams(Fptype learningRate)
+    {
+        checkParametersInitialized();
+
+        {
+            daal_dm::BlockDescriptor<Fptype> bd;
+            _learningRateTable->getBlockOfRows(0, 1, daal_dm::writeOnly, bd);
+            auto value = bd.getBlockPtr();
+            value[0] = learningRate;
+            _learningRateTable->releaseBlockOfRows(bd);
+        }
+        _solver->parameter.learningRateSequence = _learningRateTable;
+    }
+
+    // Sets the input for current iteration
+    void setInput(const daal_dm::NumericTablePtr& x, const daal_dm::NumericTablePtr& y)
+    {
+        checkParametersInitialized();
+
+        if (_nClasses == 2)
+        {
+            _logLoss->input.set(daal_solver::logistic_loss::data, x);
+            _logLoss->input.set(daal_solver::logistic_loss::dependentVariables, y);
+        }
+        else
+        {
+            _crossEntropyLoss->input.set(daal_solver::cross_entropy_loss::data, x);
+            _crossEntropyLoss->input.set(daal_solver::cross_entropy_loss::dependentVariables, y);
+        }
+    }
+
+    // Performs the single iteration of training on the data provided by setInput() method
+    void compute()
+    {
+        namespace solver = daal::algorithms::optimization_solver::iterative_solver;
+
+        checkParametersInitialized();
+
+        _solver->input.set(solver::inputArgument, _workPoint);
+        _solver->compute();
+
+        auto partialResult = _solver->getResult();
+        _workPoint = partialResult->get(solver::minimum);
+    }
+
+    // Creates final beta parameter values and resets internal algorithm state to default values
+    void finalizeCompute()
+    {
+        checkParametersInitialized();
+
+        daal_dm::BlockDescriptor<Fptype> bBetas;
+        _workPoint->getBlockOfRows(0, _nBetas*_nBetaCols, daal_dm::readOnly, bBetas);
+        if (_isIntercept)
+        {
+            _finalBetas = daal_dm::SyclHomogenNumericTable<Fptype>::create(bBetas.getBuffer(), _nBetaCols, _nBetas);
+        }
+        else
+        {
+            _finalBetas = daal_dm::SyclHomogenNumericTable<Fptype>::create(_nFeatures, _nBetas, daal_dm::NumericTableIface::doAllocate);
+            daal_dm::BlockDescriptor<Fptype> bFinalBetas;
+            _finalBetas->getBlockOfRows(0, _nBetas, daal_dm::writeOnly, bFinalBetas);
+
+            auto src = bBetas.getBlockPtr();
+            auto dst = bFinalBetas.getBlockPtr();
+
+            for (size_t i = 0; i < _nBetas; i++)
+            {
+                for (size_t j = 0; j < _nFeatures; j++)
+                {
+                    dst[i*_nFeatures + j] = src[i*_nBetaCols + j + 1];
```

Comment:
[minor] For good, this code should be on sycl.

Reply:
It's done this way for simplicity - if you insist on the change, I can apply it